azurelinuxagent/common/conf.py

```diff
@@ -164,6 +171,18 @@ def enable_firewall(conf=__conf__):
     return conf.get_switch("OS.EnableFirewall", False)
 
 
+def get_enable_firewall_period(conf=__conf__):
+    return conf.get_int("OS.EnableFirewallPeriod", 30)
```

Comment:
A lot of the new flags dont have an entry in the README, any particular reason for that?

Reply:
This should be the only one that is not in README. EnableFirewall is not there either so I am following the same

Reply:
So there is a `yml` list and then there is a description for each flag in the conf file. I found a couple that have a description in the README but have not been added to the `yml` list in the README - 
```
OS.RemovePersistentNetRulesPeriod
OS.RootDeviceScsiTimeoutPeriod
Provisioning.MonitorHostNamePeriod
```   

Missing completely from README - 
```
OS.MonitorDhcpClientRestartPeriod
```

Also I personally feel we should add the Firewall rules to the readme too, it makes more sense to be as transparent as possible especially because its an open source project.

Reply:
I'll add MonitorDhcpClientRestartPeriod to README. The other list is not in sync with README, I think it is just meant as a sample, not a comprehensive list of options. I'm not planning on listing all the missing options there.